.github/workflows/pr-go-lint.yml

```diff
@@ -20,13 +17,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-go
         with:
-          fetch-depth: 2
-
-      - uses: ./.github/actions/setup-deps
-        with:
+          go-version: 1.17
```

Comment:
remove this? as mentioned in https://github.com/vercel/turborepo/pull/1904#issuecomment-1244151642